### PR TITLE
actors graph: use regex and display actors with dialogs

### PR DESCRIPTION
**Pull request by fernflower**
Now by passing --regex option you can limit actors to display.
For example, --regex 'Authselect*|RemoveOldPAM*' would output
only authselect-pr specific actors.
Besides now actors that have dialogs will have box-shaped vertex
instead of the default oval-shaped one.
Also graph.dot file will be now overwritten instead of stopping
graph generation and exiting.

### CODE REVIEW

File: scripts/actors-models-graph.py [hunk 1]
```diff
@@ -94,17 +101,13 @@ def clusters(tags, file_handle):
     parser = argparse.ArgumentParser(description=description)
     parser.add_argument('-r', '--repo', required=True, help='path to repository with actors')
     parser.add_argument('-d', '--dot', default='graph.dot', help='path to a dot file that should be generated')
+    parser.add_argument('--regex', help='A regex that actors name should match')
```
Comment (vojtechsokol) [inline scripts/actors-models-graph.py]: Just for the sake of consistency with other arguments I would add short option for regex filtering (`-x`?).

Comment (fernflower) [reply scripts/actors-models-graph.py]: Fixed

**Comment by vojtechsokol** [comment]
LGTM.
@pirat89 Do you also want to check the PR or should I merge it straight away?

**Comment by pirat89** [comment]
You can merge it straight away.

**Comment by pirat89** [comment]
Ok I merged that already to save a time.

**Pull request merged**